### PR TITLE
CCM-816: 404 valid uri invalid request type

### PR DESCRIPTION
**Pull request by Kennedy1265**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: tests/test_not_found.py [hunk 1]
```diff
@@ -27,15 +29,33 @@ def __assert_404_error(resp, check_body=True):
 @pytest.mark.parametrize('request_path', REQUEST_PATH)
 def test_404_get(nhsd_apim_proxy_url, request_path):
     resp = requests.get(f"{nhsd_apim_proxy_url}{request_path}", headers={
-        "Accept": "*/*"
+        "Accept": "*/*",
+        "Content-Type": "application/vnd.api+json"
```
Comment (PhilipSkinner) [inline tests/test_not_found.py:33]: Content-Type not valid on GET as there isn't a body being sent with the request.

File: tests/test_not_found.py [hunk 2]
```diff
@@ -27,15 +29,33 @@ def __assert_404_error(resp, check_body=True):
...
     })
     __assert_404_error(resp)
 
 
 @pytest.mark.sandboxtest
-@pytest.mark.parametrize('request_path', REQUEST_PATH)
```
Comment (PhilipSkinner) [inline tests/test_not_found.py:36]: Any reason why you're not parameterising this anymore? Could remove the functions below.

**Review by PhilipSkinner — CHANGES_REQUESTED** [review]
Please refer to comments.

File: tests/test_not_found.py (hunk 1 above)
Comment (Kennedy1265) [reply tests/test_not_found.py]: Removed

File: tests/test_not_found.py (hunk 2 above)
Comment (PhilipSkinner) [reply tests/test_not_found.py:36]: No I think thats ok, I'll approve.

**Review by PhilipSkinner — APPROVED** [review]
Looks good, approving.

**Pull request merged**